keycloak-oauth-policy/pom.xml

```diff
@@ -12,46 +13,68 @@
   <name>apiman-plugins-keycloak-oauth-policy</name>
 
   <dependencies>
-    <!-- apiman dependencies (must be excluded from the WAR) -->
+    <!-- Non apiman deps -->
+
     <dependency>
-      <groupId>io.apiman</groupId>
-      <artifactId>apiman-gateway-engine-core</artifactId>
-      <scope>provided</scope>
+      <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Using KC 1.9.0.CR1 as they have also finally updated jackson - this avoids an issue with the annotation processing stuff where groupIds were mismatched.